unpackaged/config/qa/permissionsets/QA.permissionset

```diff
@@ -0,0 +1,380 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PermissionSet xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>A permission set for fields included in the NPSP qa_org CumulusCI flow</description>
+    <fieldPermissions>
+        <editable>true</editable>
+        <readable>true</readable>
+        <field>Account.A_User_Defined_Rollup_Field__c</field>
+    </fieldPermissions>
+        <fieldPermissions>
+        <editable>true</editable>
+        <readable>true</readable>
+        <field>Account.custom_acc_text__c</field>
+    </fieldPermissions>
+        <fieldPermissions>
+        <editable>true</editable>
+        <readable>true</readable>
+        <field>Account.UniqueId__c</field>
+    </fieldPermissions>
+        <fieldPermissions>
+        <editable>true</editable>
+        <readable>true</readable>
+        <field>%%%NAMESPACE%%%Address__c.Custom_add_date__c</field>
```

Comment:
The fields in the Gift Entry perm set have the namespace token in place before the object and before the field - should that be the case here also?

example from the GE perm set file:
`<field>%%%NAMESPACE%%%DataImportBatch__c.%%%NAMESPACE%%%Batch_Defaults__c</field>`

Reply:
The fields in this file are deployed as unmanaged metadata into a managed org for regression, so they would not have the namespace on the field.

Reply:
They'll need a token only if you expect this metadata to deploy into a namespaced scratch org, which would be the `%%%NAMESPACED_ORG%%%` token.

Reply:
Some QA orgs _are_ built namespaced, so we may want to do that.

Reply:
We did a little bit more research here. Namespaced scratch orgs automatically apply the current namespace to any field that does not have it when you're deploying metadata. This means any of the following work for deploying a permission set to a namespaced scratch org:

- `<field>%%%NAMESPACE%%%Address__c.Custom_add_date__c</field>`
- `<field>npsp__Address__c.npsp__Custom_add_date__c</field>` 
- `<field>Address__c.Custom_add_date__c</field>`

Reply:
Sounds good. 

Namespaced orgs are not 100% consistent, unfortunately. There are edge cases where that automatic resolution of the namespace doesn't work. But we don't need to worry about that unless/until it becomes a problem.

Reply:
Excellent!  Thank you @bdvorachek and @davidmreed for the research and findings!